### ISSUE TITLE
use of undeclared identifier `conn_id` while compile with flag `DEBUG=1`

### ISSUE THREAD

**Issue by xxxxxliil** [gojue/ecapture#605]
**Describe the bug**
编译带 ebpf debug 能力的 ecapture 时会看到大量的
```
kern/tc.h:248:90: error: use of undeclared identifier 'conn_id'
  248 |         debug_bpf_printk("capture packet process not found, src_port:%d, dst_port:%d\n", conn_id.src_port, conn_id.dst_port);
      |                                                                                          ^
kern/tc.h:248:108: error: use of undeclared identifier 'conn_id'
  248 |         debug_bpf_printk("capture packet process not found, src_port:%d, dst_port:%d\n", conn_id.src_port, conn_id.dst_port);
      |
```
进而导致不能编译，且返回值为 0

无论是否带 `DEBUG=1` 最前面几行输出总是有以下内容，但只有 `DEBUG=1` 才会编译失败
```
cd /usr/src/linux-source-6.10.6
whoami || exit 1
/bin/bash: 第 1 行：cd: /usr/src/linux-source-6.10.6: 没有那个文件或目录
```

**To Reproduce**
1. clone repo
2. `$ make DEBUG=1`
3. 最后一行为`make[1]: 离开目录“/home/u/Templates/ecapture/lib/libpcap”`，且以往输出没有 `go build`
4. bin/ 下无文件
**Expected behavior**
编译出带有 ebpf debug 输出的 ecapture 文件

**Screenshots**

**Linux Server/Android (please complete the following information):**

- Env:
```
---------------------------------------
eCapture Makefile Environment:
---------------------------------------
PARALLEL                 4
----------------[ from args ]---------------
CROSS_ARCH               
ANDROID                  0
DEBUG                    0
SNAPSHOT_VERSION         
---------------------------------------
HOST_ARCH                x86_64
UNAME_R                  6.10.6-zen1-1-zen
CLANG_VERSION            18
GO_VERSION               1.23
---------------------------------------
CMD_CLANG                clang
CMD_GIT                  git
CMD_GO                   go
CMD_INSTALL              install
CMD_LLC                  llc
CMD_MD5                  md5sum
CMD_PKGCONFIG            pkg-config
CMD_STRIP                llvm-strip
CMD_CC_PREFIX            
CMD_TAR                  tar
CMD_RPMBUILD             rpmbuild
CMD_RPM_SETUP_TREE       rpmdev-setuptree
---------------------------------------
VERSION_NUM              0.8.5-20240819-35cc757
LAST_GIT_TAG             0.8.5-20240819-35cc757
BPF_NOCORE_TAG           6_10_6-zen1-1-zen.0_8_5-20240819-35cc757
KERN_RELEASE             6.10.6-zen1-1-zen
KERN_BUILD_PATH          /lib/modules/6.10.6-zen1-1-zen/build
KERN_SRC_PATH            /lib/modules/6.10.6-zen1-1-zen/build
TARGET_ARCH              x86_64
GOARCH                   amd64
LINUX_ARCH               x86
LIBPCAP_ARCH             x86_64-pc-linux-gnu
AUTOGENCMD               test -f kern/bpf/x86/vmlinux.h || bpftool btf dump file /sys/kernel/btf/vmlinux format c > kern/bpf/x86/vmlinux.h
PACKAGE_VERSION          0.0.0
OUT_DEB_FILE             ./bin/ecapture_v0.0.0_linux_amd64.deb
---------------------------------------
```
- OS: arch
- Arch: amd64/x86-64
- Kernel Version: in Env
- Version: in Env

**Comment by cfc4n** [comment]
It will be fixed in #606.

**Comment by cfc4n** [comment]
The command `cd /usr/src/linux-source-6.10.6` is specific to the Ubuntu kernel; if you are using a different operating system, you will need to make some adjustments to the makefile accordingly.
